conda/environments/builddocs_py37.yml

```diff
@@ -0,0 +1,14 @@
+name: dask_cuda_docs
+channels:
+- rapidsai-nightly
+- conda-forge
+dependencies:
+# required for building docs
+- sphinx
+- sphinx_rtd_theme
+- sphinxcontrib-websupport
+- nbsphinx
+- numpydoc
+- recommonmark
+- pandoc=<2.0.0
+- pip
```

Comment:
This is fine, but we have a package that is consistently updated for all of RAPIDS docs `rapids-doc-env` specified [here](https://github.com/rapidsai/integration/blob/branch-0.14/conda/recipes/rapids-doc-env/meta.yaml)

Instead of adding yet another env file with it's own dependencies, I'd like to see this use the env pkg above. This way we're not chasing so many version updates and changes across a dozen plus repos

Reply:
Thanks @mike-wendt -- didn't know about `rapids-doc-env` in #7d5bf84 I moved everything out and used `rapids-doc-env` as we don't do anything outside of standard sphinx things for now